### ISSUE TITLE
Runtime linking not working with binary wheel distribution

### ISSUE THREAD

**Issue by tammojan** [casacore/python-casacore#171]
The following command will fail with a binary distribution, since it relies on runtime linking:
```
import casacore.tables as pt
# Get the current azimuth and elevation of the sun in Dwingeloo
pt.taql("select meas.azel('Sun', datetime(), 'DWL')")
```
This requires `libcasa_meas.so` to be in the runtime linking path. This is not the case in the current wheel distribution prototype.

**Comment by gijzelaerr** [comment]
I guess this has been resolved by https://github.com/casacore/python-casacore/pull/177 ?

**Comment by gervandiepen** [comment]
Yes; in that time runtime linking is not necessary anymore for special TaQL
functions.
However, unknown storage managers (such as LofarStMan) cannot be loaded.
Since they are part of other software, there is no way to preload them.

**Comment by gijzelaerr** [comment]
I guess we can then close this issue when it is properly documented. 

We can probably expect bug reports about this when we start distributing the binary wheels.
